src/components/select/tests/SingleSelectConnected.spec.tsx

```diff
@@ -252,7 +252,7 @@ describe('Select', () => {
             });
         });
 
-        describe('click handler', () => {
+        xdescribe('click handler', () => {

```

Comment:
Can you try changing the id of the div instead?

Reply:
tried and nothing good happen 😿